aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/libman.json

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
Because of [this](https://snyk.io/blog/bower-is-dead/) and [this](https://docs.microsoft.com/tr-tr/aspnet/core/client-side/libman/libman-vs?view=aspnetcore-3.1). Maybe i shouldn't do it? imho in following days we will be seeing libman so often.

Reply:
Yes, we may change it, but we usually use yarn. 
@ismcagdas What do you think?

Reply:
I'm not sure about LibMan. Our experiences with Bundler & Minifier was not good and as I can see same team is behind the LibMan :). I will check LibMan and than we can decide.